### PR TITLE
ITS/MFT strobe adjusted + settings to include TPC calib in async reco

### PR DESCRIPTION
**Pull request by chiarazampolli**
@shahor02 , @catalinristea 
This is needed for the runs from last night:

520259
520265
520271
520282
520283
520288
520290

After that the strobe length was changed again, but for now, no run is GOOD in bookkeeping.

### CODE REVIEW

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh [hunk 1]
```diff
@@ -82,7 +80,7 @@ export ARGS_EXTRA_PROCESS_o2_tof_reco_workflow="--use-ccdb --for-calib"
 export PVERTEXER="pvertexer.acceptableScale2=9;pvertexer.minScale2=2.;pvertexer.nSigmaTimeTrack=4.;pvertexer.timeMarginTrackTime=0.5;pvertexer.timeMarginVertexTime=7.;pvertexer.nSigmaTimeCut=10;pvertexer.dbscanMaxDist2=36;pvertexer.dcaTolerance=3.;pvertexer.pullIniCut=100;pvertexer.addZSigma2=0.1;pvertexer.tukey=20.;pvertexer.addZSigma2Debris=0.01;pvertexer.addTimeSigma2Debris=1.;pvertexer.maxChi2Mean=30;pvertexer.timeMarginReattach=3.;pvertexer.addTimeSigma2Debris=1.;pvertexer.dbscanDeltaT=24;pvertexer.maxChi2TZDebris=100;pvertexer.maxMultRatDebris=1.;pvertexer.dbscanAdaptCoef=20.;pvertexer.timeMarginVertexTime=1.3"
 export SVTX="svertexer.checkV0Hypothesis=false;svertexer.checkCascadeHypothesis=false"
 
-export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER;$MFT_STROBELGT;"
+export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER;"
```
Comment (shahor02) [inline DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh:83]: Note that until https://github.com/AliceO2Group/AliceO2/pull/9352 is deployed, for non-standard strobe settings one should have it also in the p.vertexer settings, i.e.
```
export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER;ITSAlpideParam.roFrameLengthInBC=$ITS_STROBE;MFTAlpideParam.roFrameLengthInBC=$MFT_STROBE;"
```

Comment (chiarazampolli) [reply DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh:83]: Ok, I can add it for safety.

**Comment by shahor02** [comment]
@chiarazampolli yes, the ITS is set in the prodtests/full-system-test/workflow-setup.sh if it is not defined, MFT is used only if it is set.
So, you can simply set 
```
export ITS_STROBE="198"
export MFT_STROBE="198"
```

**Comment by chiarazampolli** [comment]
Yes, this is what I did. Thanks for the feedback!

**Pull request merged**